dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/ValueType.java

```diff
@@ -222,10 +222,14 @@ public boolean isAggregatable( AggregationType aggregationType )
         {
             return false;
         }
+
         if ( this == TEXT )
         {
-            return aggregationType == AggregationType.NONE;
+            return Set.of(
```

Comment:
Not a big deal, but I believe would be nicer to extract this "Set" into a constant.

Reply:
Agreed, it is not ideal to create that set on every invocation.